app/src/main/java/com/example/cattlelog/MainActivity.kt

```diff
@@ -86,15 +106,59 @@ class MainActivity : AppCompatActivity() {
             cattleList?.let { adapter.setCattleList(it) }
         })
 
+    }
 
-//        recyclerView = findViewById<RecyclerView>(R.id.herdList).apply {
-//            setHasFixedSize(true)
-//            layoutManager = viewManager
-//            adapter = viewAdapter
-//        }
+    private fun checkDatabaseVersion() {
```

Comment:
I would rename this to `getDatabaseVersion()` and have it return the version. See your comment down below where you say, `compare the string stored locally against the result from the `checkDatabaseVersion()` function.`